sunsynk/definitions.py

```diff
@@ -146,18 +145,67 @@
 ###########
 _SENSORS += (
     Sensor(200, "Control Mode"),
-    Sensor(201, "Equalization voltage", VOLT, 0.01),
-    Sensor(202, "Absorption voltage", VOLT, 0.01),
     Sensor(230, "Grid Charge Battery current", AMPS, -1),
     Sensor(232, "Grid Charge enabled", "", -1),
     Sensor(312, "Battery charging voltage", VOLT, 0.01),
     Sensor(603, "Bat1 SOC", "%"),
     Sensor(611, "Bat1 Cycle"),
 )
-BATTERY_SHUTDOWN_CAPACITY = RWSensor(217, "Battery Shutdown Capacity", "%")
-BATTERY_RESTART_CAPACITY = RWSensor(218, "Battery Restart Capacity", "%")
-BATTERY_LOW_CAPACITY = RWSensor(219, "Battery Low Capacity", "%")
-_SENSORS += (BATTERY_SHUTDOWN_CAPACITY, BATTERY_RESTART_CAPACITY, BATTERY_LOW_CAPACITY)
+
+# Absolute min and max voltage based on Deye inverter
+MIN_VOLTAGE = 41
+MAX_VOLTAGE = 60
```

Comment:
These were obtained by checking out how far the values for the  Equalization, Absorption and Float voltage sensors can go on my Deye inverter.